README.rst

```diff
@@ -31,7 +31,36 @@ You can use ``pip`` to install codespell with e.g.::
 Usage
 -----
 
-Check usage with ``codespell -h``. There are a few command line options.
+For more in depth info please check usage with ``codespell -h``. 
+
+Some noteworthy flags::
+
+    codespell -w, --write-changes
+
+The -w flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is equated with doing a dry run. Recommended to run this with the ``-i`` or ``--ineractive`` flag.:: 
+
+    codespell -S, --skip=
+
+Comma-separated list of files to skip. It accepts globs as well.  Example:
+ 
+to skip .eps & .txt files, invoke ``codespell --skip="*.eps,*.txt"``  
+
+to skip directories, invoke ``codespell --skip="./src/3rd-Party,./src/Test"`` ::
+
+    codespell -I FILE, --ignore-words=FILE
+
+This flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. Format of whitelist file is one word per line. Invoke ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. 
```

Comment:
Format of whitelist -> The format of the whitelist

Reply:
👍